maintenance_equipment_status/views/maintenance_equipment_views.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- equipment : views -->
+    <record id="hr_equipment_view_form" model="ir.ui.view">
+        <field name="name">equipment.form</field>
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form"/>
+        <field name="arch" type="xml">
+            <sheet position="before">
+                <header>
+                    <field name="status_id" widget="statusbar" clickable="True"
+                           domain="['|', ('category_ids', 'in', [category_id]), ('category_ids', '=', False)]"/>
+
+                </header>
+            </sheet>
+        </field>
+    </record>
+
+    <record id="hr_equipment_view_tree" model="ir.ui.view">
+        <field name="name">equipment.tree</field>
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_tree"/>
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="status_id"/>
+            </field>
+        </field>
+    </record>
+
+    <record id="hr_equipment_view_kanban" model="ir.ui.view">
+        <field name="name">equipment.kanban</field>
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_kanban"/>
+        <field name="arch" type="xml">
+            <field name="activity_state" position="after">
+                <field name="status_name"/>
+            </field>
+            <xpath expr="//div[@class='oe_kanban_bottom_left']" position="inside">
```

Comment:
There is a warning regarding this, it is better to use //div[hasclass('oe_kanban_bottom_left')]

Reply:
change if by the comment of @jarroyomorales and it will be ready for merge :green_apple: